x-pack/plugins/gis/public/shared/layers/sources/es_geohashgrid_source/update_source_editor.js

```diff
@@ -49,14 +50,17 @@ export class UpdateSourceEditor extends Component {
     this.props.onChange({ propName: 'metrics', value: metrics });
   };
 
+  _onResolutionChange =    (e) => {
+    this.props.onChange({ propName: 'resolution', value: e });
+  };
+
   _renderMetricsEditor() {
 
     const metricsFilter = (this.props.renderAs === RENDER_AS.HEATMAP) ?  ((metric) => {
       //these are countable metrics, where blending heatmap color blobs make sense
       return ['count', 'sum'].includes(metric.value);
     }) : null;
-    const allowMultipleMetrics = (this.props.renderAs === RENDER_AS.HEATMAP) ? false : true;
-
+    const allowMultipleMetrics = (this.props.renderAs !== RENDER_AS.HEATMAP);
```

Comment:
Nice fix. Don't even need parentheses

`const allowMultipleMetrics = this.props.renderAs !== RENDER_AS.HEATMAP;`